app/uk/gov/hmrc/transitmovementsauditing/controllers/ErrorTranslator.scala

```diff
@@ -49,6 +49,7 @@ trait ErrorTranslator {
   implicit val conversionError = new Converter[ConversionError] {
 
     def convert(conversionError: ConversionError): PresentationError = conversionError match {
+      case ConversionError.FailedConversion(_)     => PresentationError.internalServiceError()
       case ConversionError.UnexpectedError(_, thr) => PresentationError.internalServiceError(cause = thr)
```

Comment:
are we logging the cause of internal service errors?

Reply:
No, and I'm always a little concerned about doing so because we don't know what might get logged...

Reply:
If we aren't logging the exception, how do we identify the cause of the internal service error?